packages/sitecore-jss-nextjs/src/index.ts

```diff
@@ -76,6 +76,11 @@ export {
   GraphQLSitemapServiceConfig,
 } from './services/graphql-sitemap-service';
 
+export {
+  GraphQLSitemapService as GraphQLSiteSitemapService,
+  GraphQLSitemapServiceConfig as GraphQLSiteSitemapServiceConfig,
+} from '@sitecore-jss/sitecore-jss/site';
+
```

Comment:
Oh, I forgot we already have a `GraphQLSitemapService`... Let's name yours `GraphQLSitemapXmlService`